src/coreComponents/constitutive/solid/ElasticIsotropic.hpp

```diff
@@ -143,23 +143,6 @@ class ElasticIsotropicUpdates : public SolidBaseUpdates
     return m_shearModulus[k];
   }
 
-  // TODO: confirm hyper stress/strain measures before activatiing
```

Comment:
Can we leave the `hyperUpdate` code here, but still commented out?  We will likely activate this in the future, and we don't want to lose the existing implementation from GEOS-classic.  It should not impact this PR to leave it here.

Reply:
Done, I put the `hyperUpdate` back